### ISSUE TITLE
Feature Request: add a shortcut to add product to the shopping list

### ISSUE THREAD

**Issue by Choukajohn** [patzly/grocy-android#658]
Hi,

It would be useful if you could add a shortcut to add products to the shopping list.
This shortcut would be available in the stock overview depending on the stock limits.

Anyway, thank you for this great app!

**Comment by patzly** [comment]
What do you exactly mean with shortcut? A shortcut is normally a launcher shortcut which shows up when you long-press the app icon in your launcher. This shortcut can be already enabled in behavior settings.
If you just mean a quick way to add a product from stock overview to the shopping list, this is also already available: tap on the product, the product overview opens. Then tap on the three dots and select "Add to shopping list".

**Comment by Choukajohn** [comment]
Sorry for the misunderstanding.
I mean add a shortcut when you click on the product from the stock overview to avoid the Clic on the three dots.
Actually, clicking everytime on the three dots is not very optimised when you want to add multiple products to the shopping list.

**Comment by patzly** [comment]
Ok now I understand your problem, but do you already have tried the following prozedure? With this I never need such a shortcut.

- Set the required minimum stock amount for the products in their product options
- Go to shopping list > bottom menu > Add missing products

With this prozedure there is no need to go through multiple products in stock overview. If you don't want to change the minimum stock amount, you can instead create a recipe with the required incredients, then you can tap on "Add to shopping list".

**Comment by patzly** [comment]
Yeah you can go to the master product page of the product  > Optional > Active :)

**Comment by Choukajohn** [comment]
Well thanks for the tip!
But for several reasons, like to have a look on some differents products out of stock and to choose which one I will buy or not this time, I prefer to use the "add to shopping list" feature from the stock overview and not the "add all missing products" function.

Nevermind, if it is too much work or not in your plans to add this feature, I understand.
Thanks again for your help and work!
